addons/modules/functions/fnc_moduleTracers.sqf

```diff
@@ -0,0 +1,75 @@
+#include "script_component.hpp"
+/*
+ * Author: Ampersand
+ * Zeus module function to shoot tracers.
+ *
+ * Arguments:
+ * 0: Display <DISPLAY>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [DISPLAY] call zen_modules_fnc_moduleTracers
+ *
+ * Public: No
+ */
+
+ params ["_display"];
+
+ private _logic = GETMVAR(BIS_fnc_initCuratorAttributes_target,objNull);
+ _display closeDisplay IDC_CANCEL; // Close helper display
+
+ // Need to delay dialog creation by one frame to avoid weird input blocking bug
+ [{
+    params ["_logic"];
+
+    // Default values: green tracers, 10-20s between bursts
+    private _tracersParams = _logic getVariable [QGVAR(tracersParams), [east, 10, 20, 0.05, "", "", 0, ""]];
+    _tracersParams params ["_side", "_min", "_max", "_dispersion", "_weapon", "_magazine", "_targetType", "_target"];
+
+    ["str_a3_cfgvehicles_moduletracers_f_0", [
+        ["SIDES",  ELSTRING(attributes,ChangeSide), _side, true],
+        ["SLIDER", "str_3den_attributes_timeout_titlemin_text", [0, 120, _min, 0], true],
+        ["SLIDER", "str_3den_attributes_timeout_titlemax_text", [0, 120, _max, 0], true],
+        ["SLIDER", ELSTRING(attributes,Dispersion), [0.001, 0.300, _dispersion, 3], true],
```

Comment:
I think a range of options like "Very Low", "Low", "Medium", "High", and "Very High" might be more immediately clear than a numeric input from 0.001 to 0.3.

Reply:
3d94e66